src/trezor.ts

```diff
@@ -462,25 +428,22 @@ export class TrezorGetMetadata extends TrezorInteraction {
  * You may want to use `TrezorExportPublicKey` or
  * `TrezorExportExtendedPublicKey` directly.
  *
- * @extends {module:trezor.TrezorInteraction}
  * @example
- * import {MAINNET} from "unchained-bitcoin";
+ * import {Network} from "unchained-bitcoin";
  * import {TrezorExportHDNode} from "unchained-wallets";
- * const interaction = new TrezorExportHDNode({network: MAINNET, bip32Path: "m/48'/0'/0'/2'/0"});
+ * const interaction = new TrezorExportHDNode({network: Network.MAINNET, bip32Path: "m/48'/0'/0'/2'/0"});
  * const node = await interaction.run();
  * console.log(node); // {publicKey: "", xpub: "", ...}
  *
  */
 export class TrezorExportHDNode extends TrezorInteraction {
-  /**
-   * Requires a BIP32 path to the node to export as well as which network.
-   *
-   * @param {object} options - options argument
-   * @param {string} options.network - bitcoin network
-   * @param {string} bip32Path - the BIP32 path for the HD node
-   * @param {boolean} includeXFP - return xpub with root fingerprint concatenated
-   */
-  constructor({ network, bip32Path, includeXFP }) {
+  bip32Path: string;
+
+  includeXFP: boolean;
+
+  bip32ValidationErrorMessage: any;
```

Comment:
Would this not need to be a string?

Reply:
Changed in 35668a8805582ecc715ddc5e5214ac4de91c22ff